modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -996,11 +1012,19 @@ public Expression textAnchor() {
      * @return AnchorPoint defined by "text-anchor".
      */
     public AnchorPoint anchorPoint() {
```

Comment:
Trying to update, having some difficulties with the Mapbox styles version, as it does not seem to be versioned stand-alone anymore, it's now just part of the JS package, see:
https://github.com/mapbox/mapbox-gl-style-spec
https://github.com/mapbox/mapbox-gl-js/tree/master/src/style-spec

The files are versioned at 8, fixed, no matter what the actual version of the spec is... bizzarre...

The changelog in the new repo does not even contain version 0.10.0 and does not mention text-anchor:
https://github.com/mapbox/mapbox-gl-js/blob/master/src/style-spec/CHANGELOG.md

However doing a blame on the [spec file](https://github.com/mapbox/mapbox-gl-js/blame/master/src/style-spec/reference/v8.json) I've found the change that implemented it:
https://github.com/mapbox/mapbox-gl-js/pull/4997

The issue associated does not help, does not have a milestone... dunno, not sure what to do, in this situation it might make sense to just drop the mapbox spec versions, they are hard to find and not reflected in the style files anyways...

Reply:
I mostly just care about tracking whether (and which version of) GeoTools supports a specific feature in the spec, so if you want to ignore or drop the mapbox spec version that's fine with me.

Reply:
Ok... hum... we want to backport these fixes, but not entirely sure what version they will land on. I'll tentatively write 22.2 in the docs, ok?

Reply:
Seems reasonable to me; release notes for the initial minor versions should at least include these JIRA ticket numbers, so the major version is more important, for when someone looks at the spec in another year or so to try and figure out what is currently implemented.

As for the Mapbox version I do notice that on the mapbox spec page shows 0.10.0 under MapBox GL JS for a lot of the properties that have been supported a while, which is the same as what we have under MapBox version for those properties. So you could probably just go with 0.35.0 to match their spec - https://docs.mapbox.com/mapbox-gl-js/style-spec/#layout-symbol-text-offset